modules/nf-core/gmmdemux/tests/nextflow.config

```diff
@@ -0,0 +1,17 @@
+params{
+    // Optional parameters for GMM-demux - Default values
+    threshold = 0.8
+    num_cells = 35685
+    examine = "example_cell_types.txt"
+    ambiguous = 0.05
```

Comment:
Are these default values that are set by the tool?

Reply:
threshold yes, I was using the number of cells of the original dataset from the tool, I just changed it to the number of cells of the dataset from nf-core :)